maas-region/src/charm.py

```diff
@@ -55,6 +56,12 @@
 class MaasRegionCharm(ops.CharmBase):
     """Charm the application."""
 
+    _TLS_MODES = [
+        "",
+        "termination",
+        "passthrough",
```

Comment:
either remove this missing mode or raise an exception when the user selects it. we should not allow the user to set an option that does nothing and fails silently.